projects/observability/src/shared/graphql/request/handlers/explore/explore-graphql-query-builder.service.ts

```diff
@@ -108,4 +111,14 @@ export class ExploreGraphqlQueryBuilderService {
       this.specBuilder.exploreSpecificationForAttributeExpression(expression)
     );
   }
+
+  private buildEntityContextOptionsArgument(entityContextOptions?: EntityContextOptions): GraphQlArgument[] {
+    if (!isNil(entityContextOptions) && !isNil(entityContextOptions.includeNonLiveEntities)) {
```

Comment:
```suggestion
    if (!isNil(entityContextOptions?.includeNonLiveEntities)) {
```

Reply:
Can't do this because the compiler will say this 
![Screenshot 2023-08-04 at 6 10 53 PM](https://github.com/hypertrace/hypertrace-ui/assets/78212931/ab439c15-96cb-4f6f-87b6-561f33c275e8)

Reply:
We would have to use assertion